### ISSUE TITLE
Feature Request: Aeroponics Pump timer

### ISSUE THREAD

**Issue by TemporalFixation** [fruxefarms/FruxePi#64]
Hi, I really like the project and the work you've been putting into it shows!

I was wondering if there was a way to have a pump cycle on for a variable of time, and then off for another variable?

For example, with some aeroponics projects, I run my pump for 1 min every 5 minutes. Currently, I use an arduino sketch on the relay for them pump, but if I could manage it with Fruxe through the web app, it would be awesome!

**Comment by fruxefarms** [comment]
Yes indeed! That sounds like a great idea, and borders on similar requests for the pump feature. There is a new version coming mid-November and will incorporate a format like this into the scheduler. Thank you again for your ideas and support!
